openupgrade_scripts/scripts/product/16.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,29 @@
+---Models in module 'product'---
+new model product.tag
```

Comment:
It seems that odoo integrates now oca features présent in https://github.com/OCA/product-attribute/tree/14.0/product_template_tags

If yes some migrations scripts could be added. (Not in the scope of this PR)
Cc @ivantodorovich 
@patrickrwilson